src/containers/Trade.svelte

```diff
@@ -32,7 +32,7 @@
   import { showToast } from '../utils/toast';
   import LoadingModal from '../components/LoadingModal.svelte';
   import { getExplorerForNetwork } from '../utils/explorer';
-  import { detectProvider, MarinaProvider } from 'marina-provider';
+  import type { NetworkString } from 'ldk';
```

Comment:
Note that tdex-sdk exports all the LDK types, so u can import all the LDK types from tdex-sdk.

Reply:
Commit c352eda fixes this.